sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/postgres/PostgresArrayTests.java

```diff
@@ -44,10 +37,11 @@ public void testSplit() {
                 " {}\n" +
                 "(1 row)\n" +
                 "\n" +
+                // Semantics different from Postgres
                 "select split('1|2|3', NULL);\n" +
                 " split \n" +
                 "-----------------\n" +
-                " {1,|,2,|,3}\n" +
+                " NULL\n" +
```

Comment:
Some of these are quite the change in behavior with just a minor version bump? What's going on?

Reply:
this test was disabled before. But indeed, the behavior is different from Postgres, where I took the test from.